circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/recursion/aggregation_state/aggregation_state.hpp

```diff
@@ -109,4 +110,4 @@ template <typename NCT> std::ostream& operator<<(std::ostream& os, aggregation_s
 
 } // namespace recursion
 } // namespace stdlib
-} // namespace proof_system::plonk
\ No newline at end of file
+} // namespace proof_system::plonk
```

Comment:
Huh?

Reply:
I had intentionally triggered a build - thought I reverted the file